### PR TITLE
Issue 4875 - CLI - Add some verbosity to installer

### PR DESCRIPTION
**Pull request by mreynolds389**
Description:  

Previously the installer would basically say "Starting" and "Finished".  If a step would run into a problem it is difficult to narrow down what is going wrong.  So add a little more output during the installation.

relates: https://github.com/389ds/389-ds-base/issues/4875

### CODE REVIEW

File: src/lib389/lib389/instance/setup.py [hunk 1]
```diff
@@ -948,6 +951,7 @@ def _install_ds(self, general, slapd, backends):
         # Create the backends as listed
         # Load example data if needed.
         for backend in backends:
+            self.log.info("Create database backends ...")
```
Comment (Firstyear) [inline src/lib389/lib389/instance/setup.py]: Should be one line up (not in the loop)

**Review by Firstyear — APPROVED** [review]
Besides that nit, looks good.

File: src/lib389/lib389/instance/setup.py (hunk 1 above)
Comment (mreynolds389) [reply src/lib389/lib389/instance/setup.py]: hehe, whoops, I'll get that fixed.

**Review by droideck — APPROVED** [review]
Looks good to me too!

File: src/lib389/lib389/instance/setup.py (hunk 1 above)
Comment (mreynolds389) [reply src/lib389/lib389/instance/setup.py]: I decided to keep it in the loop, but display which backend is being created

**Pull request merged**